example_localVerify_test.go

```diff
@@ -0,0 +1,155 @@
59, 216, 152, 222, 163, 5, 136, 222, 34, 164, 41, 163, 51, 125, 149, 1, 238, 84, 83, 25, 119, 68, 185, 144, 41, 45, 180, 206, 255, 114, 85, 113, 31, 95, 126, 237, 52, 55, 85, 26, 0, 255, 88, 24, 73, 82, 183, 130, 67, 37, 128, 61, 51, 231, 127, 66, 218, 62, 30, 34, 44, 19, 66, 163, 250, 55, 103, 192, 134, 40, 188, 243, 206, 150, 188, 138, 56, 183, 197, 111, 250, 245, 24, 74, 187, 159, 173, 241, 21, 156, 252, 222, 44, 150, 53, 166, 93, 176, 158, 77, 221, 129, 241, 116, 234, 100, 108, 22, 194, 106, 83, 132, 4, 50, 83, 141, 2, 170, 120, 6, 1, 46, 128, 13, 53, 77, 14, 6, 201, 102, 168, 112, 146, 5, 49, 39, 133, 186, 137, 28, 222, 144, 43, 33, 116, 245, 39, 14, 151, 146, 240, 51, 84, 44, 78, 93, 20, 99, 155, 34, 24, 176, 31, 143, 248, 210, 127, 211, 135, 245, 82, 211, 60, 115, 146, 243, 243, 116, 41, 174, 120, 27, 105, 111, 46, 99, 110, 99, 102, 46, 110, 111, 116, 97, 114, 121, 46, 115, 105, 103, 110, 105, 110, 103, 65, 103, 101, 110, 116, 110, 78, 111, 116, 97, 116, 105, 111, 110, 47, 49, 46, 48, 46, 48, 88, 181, 123, 34, 116, 97, 114, 103, 101, 116, 65, 114, 116, 105, 102, 97, 99, 116, 34, 58, 123, 34, 109, 101, 100, 105, 97, 84, 121, 112, 101, 34, 58, 34, 97, 112, 112, 108, 105, 99, 97, 116, 105, 111, 110, 47, 118, 110, 100, 46, 100, 111, 99, 107, 101, 114, 46, 100, 105, 115, 116, 114, 105, 98, 117, 116, 105, 111, 110, 46, 109, 97, 110, 105, 102, 101, 115, 116, 46, 118, 50, 43, 106, 115, 111, 110, 34, 44, 34, 100, 105, 103, 101, 115, 116, 34, 58, 34, 115, 104, 97, 50, 53, 54, 58, 99, 48, 100, 52, 56, 56, 97, 56, 48, 48, 101, 52, 49, 50, 55, 99, 51, 51, 52, 97, 100, 50, 48, 100, 54, 49, 100, 55, 98, 99, 50, 49, 98, 52, 48, 57, 55, 53, 52, 48, 51, 50, 55, 50, 49, 55, 100, 102, 97, 98, 53, 50, 50, 54, 50, 97, 100, 99, 48, 50, 51, 56, 48, 99, 34, 44, 34, 115, 105, 122, 101, 34, 58, 53, 50, 56, 125, 125, 89, 1, 0, 84, 33, 131, 192, 227, 144, 127, 136, 92, 165, 154, 90, 22, 179, 243, 105, 77, 91, 33, 122, 231, 157, 37, 190, 167, 157, 114, 180, 93, 30, 58, 5, 139, 56, 1, 224, 151, 85, 86, 18, 68, 50, 182, 6, 175, 173, 3, 208, 187, 5, 113, 232, 120, 35, 28, 170, 112, 7, 107, 202, 172, 101, 31, 223, 112, 198, 189, 22, 177, 15, 35, 163, 229, 241, 5, 68, 255, 164, 17, 50, 68, 254, 14, 185, 254, 244, 36, 133, 199, 49, 228, 232, 37, 61, 75, 91, 209, 177, 69, 88, 53, 200, 155, 201, 232, 231, 185, 81, 4, 68, 246, 26, 57, 26, 231, 125, 55, 57, 192, 139, 56, 95, 23, 170, 166, 185, 171, 46, 41, 140, 48, 217, 254, 187, 205, 148, 105, 219, 92, 231, 187, 91, 29, 182, 60, 86, 30, 23, 16, 163, 254, 250, 16, 73, 36, 219, 208, 19, 173, 23, 229, 165, 138, 33, 45, 145, 156, 225, 28, 226, 127, 83, 236, 183, 77, 209, 179, 139, 212, 234, 12, 214, 85, 184, 137, 49, 76, 135, 55, 220, 203, 245, 102, 23, 195, 235, 57, 43, 69, 75, 210, 229, 149, 125, 201, 235, 213, 116, 91, 1, 219, 152, 61, 247, 114, 59, 85, 4, 48, 133, 227, 133, 23, 69, 104, 11, 64, 102, 196, 235, 38, 154, 86, 103, 113, 17, 64, 70, 42, 164, 148, 120, 119, 252, 26, 158, 214, 161, 159, 216, 104, 89, 179, 197, 10, 255}
```

Comment:
> This envelope is huge. Since it is an example not a strict test, can we use `Signer.Sign()` to create an envelope?

I actually changed to the current version from using `Signer.Sign()` on a second thought:
1. I think it might be better if we completely separate the Sign and Verify logics in these examples so that users will be clearer on the requirements for Sign and Verify respectively. Especially for a new beginner, Sign requires the certificate, but not the trust store/trust policy. I found this part has confused quiet number of users so far :)
2. For a user already having a signature envelope and a certificate, he could easily replace them into the current version and run the example in go playground. Using Signer.Sign() might need a fair amount of changes on the code? If the user does not have a signature or the certificate, then the localSign example is a pretty good starting point for him.

Reply:
If we still need this large block of binary data in the end, please encode it into a base64 string or a PEM block.
```
-----BEGIN EXAMPLE SIGNATURE ENVELOPE-----
0oRYnqUBOCQCgXgcaW8uY25jZi5ub3Rhcnkuc2lnbmluZ1NjaGVtZQN4K2FwcGxp
Y2F0aW9uL3ZuZC5jbmNmLm5vdGFyeS5wYXlsb2FkLnYxK2pzb254GmlvLmNuY2Yu
bm90YXJ5LnNpZ25pbmdUaW1lwRpju22GeBxpby5jbmNmLm5vdGFyeS5zaWduaW5n
U2NoZW1la25vdGFyeS54NTA5ohghgVkDRDCCA0AwggIooAMCAQICAVEwDQYJKoZI
hvcNAQELBQAwTjELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAldBMRAwDgYDVQQHEwdT
ZWF0dGxlMQ8wDQYDVQQKEwZOb3RhcnkxDzANBgNVBAMTBmFscGluZTAgFw0wMDA4
MjkxMzUwMDBaGA8yMTIzMDgyOTEzNTAwMFowTjELMAkGA1UEBhMCVVMxCzAJBgNV
BAgTAldBMRAwDgYDVQQHEwdTZWF0dGxlMQ8wDQYDVQQKEwZOb3RhcnkxDzANBgNV
BAMTBmFscGluZTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAKHIN6hL
MjQwy3wfDhw+HYYvjNTTytQLMuTV/OHPoL2nGbqDy08JqTn5upz7exjzwfRu0usc
YRTW0cU2H2FIyvpGgo9/F4wUX+ZRnsG0iSlZMUPNv2sVO9HHkltyaWs62oHjfSVE
2fyX4uDH54qSE8HJjKIoo/Hhqx7JI8lcmWyXdFjDCkQ1lYSz1IjzFVrf+He2LWSL
c2nGkCrV5l4LEwk1qSKJbN4H7TWI60KDLFHpVHQ/LzgFjfSdvP0sgvrkofytSn8l
JW6rn5+HYvAxAcZ7T+cJ12GyS9Y7Y7FIBMQFY0MU9cyOfV9+pt7d2CqgkIdXLndN
i+aJzm2Os4+ezekCAwEAAaMnMCUwDgYDVR0PAQH/BAQDAgeAMBMGA1UdJQQMMAoG
CCsGAQUFBwMDMA0GCSqGSIb3DQEBCwUAA4IBAQAvAIwS8uxLEaYuqILgnRUm+p0R
o7xdezfm+pg297AEZwfE9acb8009usmvgMlkfo46HRMGzCEvtd5Vfvak9i8KpWzl
DWOdfT2thkUWO9iY3qMFiN4ipCmjM32VAe5UUxl3RLmQKS20zv9yVXEfX37tNDdV
GgD/WBhJUreCQyWAPTPnf0LaPh4iLBNCo/o3Z8CGKLzzzpa8iji3xW/69RhKu5+t
8RWc/N4sljWmXbCeTd2B8XTqZGwWwmpThAQyU40CqngGAS6ADTVNDgbJZqhwkgUx
J4W6iRzekCshdPUnDpeS8DNULE5dFGObIhiwH4/40n/Th/VS0zxzkvPzdCmueBtp
by5jbmNmLm5vdGFyeS5zaWduaW5nQWdlbnRuTm90YXRpb24vMS4wLjBYtXsidGFy
Z2V0QXJ0aWZhY3QiOnsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tl
ci5kaXN0cmlidXRpb24ubWFuaWZlc3QudjIranNvbiIsImRpZ2VzdCI6InNoYTI1
NjpjMGQ0ODhhODAwZTQxMjdjMzM0YWQyMGQ2MWQ3YmMyMWI0MDk3NTQwMzI3MjE3
ZGZhYjUyMjYyYWRjMDIzODBjIiwic2l6ZSI6NTI4fX1ZAQBUIYPA45B/iFylmloW
s/NpTVsheuedJb6nnXK0XR46BYs4AeCXVVYSRDK2Bq+tA9C7BXHoeCMcqnAHa8qs
ZR/fcMa9FrEPI6Pl8QVE/6QRMkT+Drn+9CSFxzHk6CU9S1vRsUVYNcibyejnuVEE
RPYaORrnfTc5wIs4XxeqprmrLimMMNn+u82Uadtc57tbHbY8Vh4XEKP++hBJJNvQ
E60X5aWKIS2RnOEc4n9T7LdN0bOL1OoM1lW4iTFMhzfcy/VmF8PrOStFS9LllX3J
69V0WwHbmD33cjtVBDCF44UXRWgLQGbE6yaaVmdxEUBGKqSUeHf8Gp7WoZ/YaFmz
xQr/
-----END EXAMPLE SIGNATURE ENVELOPE-----
```

Reply:
Oh yeah, encoding to pem block looks much better than hard coding the byte slice into the example.
I also created the generateExampleSignatureEnvelope() func as suggested so that the signature won't appear in the main example logic.